.github/workflows/protocol.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Protocaol'
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - gitlab/protocols/**
+  pull_request:
+    branches:
+      - main
+    paths:
+      - gitlab/protocols/**
+
+jobs:
+  protocol:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Fail
+      run: /bin/false
```

Comment:
Perhaps we can use one of our existing test suites if we write tests for this.

See also my comment on your other PR - if it maybe makes sense to do this at the backend level, and hide some of this from the client.

Reply:
I can't figure out how to implement a test which will identify the breaking change if a contributor would update the protocol.  The maintainer will have to manually watch for protocol change and be alerted. That is what this PR was about. Do you have ideas how to set it up?